packages/react-components/react-combobox/stories/Dropdown/DropdownTruncation.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { Dropdown, makeStyles, Option, shorthands, useId } from '@fluentui/react-components';
+import type { DropdownProps } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  root: {
+    // Stack the label above the field with a gap
+    display: 'grid',
+    gridTemplateRows: 'repeat(1fr)',
+    justifyItems: 'start',
+    ...shorthands.gap('2px'),
+    maxWidth: '200px',
+  },
+  listbox: {
+    maxHeight: '200px',
+  },
+  valueText: {
+    overflowX: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+  },
+});
+
+export const TruncatedValue = (props: Partial<DropdownProps>) => {
+  const dropdownId = useId('dropdown');
+  const options = [
+    'Cat',
+    'Caterpillar',
+    'Corgi',
+    'Chupacabra',
+    'Dog',
+    'Ferret',
+    'Fish',
+    'Fox',
+    'Hamster',
+    'Snake',
+    'Screaming hairy armadillo (Chaetophractus vellerosus)',
+  ];
+  const styles = useStyles();
+
+  const placeholder = 'Select an animal';
+  const [value, setValue] = React.useState(placeholder);
```

Comment:
Perhaps have the default value be the long one so the example shows truncation by default?

Reply:
ohhhh, good idea. Dunno why I didn't think of doing that 🤦🏻‍♀️